src/interfaces/filing-history-list-state-interface.ts

```diff
@@ -0,0 +1,15 @@
+import { ApiFilingIF } from '@/interfaces'
```

Comment:
Basically, these are the properties that were in FilingHistoryList.vue but are now used by all different sub-components.